docs/conf.py

```diff
@@ -0,0 +1,17 @@
+project = "open_petro_elastic"
+copyright = "2021, Equinor"
```

Comment:
Is this correct? I guess it is at the moment, but as this will be open sourced, will we use a CLA here?

Reply:
I simply reused the copyright string from pypdm. I am not sure what it should be.

Reply:
I dont know either, just know there has been some discussions around it.